### PR TITLE
Add play-nitrox.net

### PR DESCRIPTION
**Pull request by derdj2301**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/nitrox/manifest.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "direct_ip": "play-nitrox.net",
+    "social": {
+      "tiktok": "nitroxnet",
+      "instagram": "nitroxnet",
+      "teamspeak": "ts.nitroxnetwork.eu",
+      "discord": "https://discord.gg/MU8kM7vYpJ"
+    },
+    "discord": {
+      "server_id": 966020442571808848,
+      "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/nitrox/manifest.json:13]: Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

**Comment by derdj2301** [comment]
Is it good now? @JakobBruening

**Review by JakobBruening — APPROVED** [review]
Great, thank you!

**Comment by derdj2301** [comment]
Thank you!

**Pull request merged**